backend/lib/api/rest-api/serviceRunner.mjs

```diff
@@ -7,17 +11,45 @@ export async function runUseCase(useCaseClass, { context = {}, params = {} }) {
 export function makeUseCaseRunner(
     useCaseClass,
     paramsBuilder,
+    params,
     render = renderPromiseAsJson
 ) {
     return async function useCaseRunner(req, res, next) {
+        let context = undefined;
+        if (params?.withToken){
+            context = await validateJwt(req, res)
+            if (!context) return ;
+        }
         const resultPromise = runUseCase(useCaseClass, {
-            params : paramsBuilder(req, res)
+            params : paramsBuilder(req, res),
+            context,
         });
-
         return render(req, res, resultPromise, next);
     };
 }
 
+async function validateJwt(req, res) {
+    try {
+        const token = getToken(req);
+        const userData = await jwt.verify(token, config.tokenEmailKey);
+
+        const isValid = await User.findByPk(userData.id);
```

Comment:
if you use "isValid" then the expected value should be true/false
so rename the variable to "user"